ols/src/llms/providers/provider.py

```diff
@@ -246,10 +246,10 @@ def _override_params(self, params: dict[Any, Any]) -> dict[Any, Any]:
         updated_params = {**self.default_params, **params}
 
         # config params overrides everything
-        if config.dev_config.llm_params:
+        if config.config.dev_config.llm_params:
```

Comment:
I'm still thinking that by using "better" import strategy, just the `import` statement might change,nothing else in such modules

Reply:
Sure, we can do `from ols.utils.config import config` in main `__init__.py`, then we would just import `from ols import config` and the rest of the code should be untouched. But it would mess it up at this point as some parts of the code accessing the config module and others config holding `Config`. But yes, that is the goal.